### ISSUE TITLE
Dropdown with pullRight not displayed correctly when menu changes

### ISSUE THREAD

**Issue by lukezavrel** [react-bootstrap/react-bootstrap#5127]
## Describe the bug
When `Dropdown` with `pullRight` and menu changes, menu item is not aligned properly 

## To Reproduce
1. Use Dropdown with pullRight
2. Have some initial menuItems and change them (in example by timeout, in our case by rest response)
3. menuItems are not re-aligned, translate3d css property is not changed

## Reproducible Example
https://codesandbox.io/s/sweet-lewin-pls1v
Click on button, after 1s the displayed menuItem changes and is not displayed properly

## Expected behavior
Dropdown should react to menu change. The example is now set to change menu to its initial value when dropdown is closed, but when this functionality is commented, then after rerender, the menu shows correctly. 

## Screenshots
This is how it is displayed
![image](https://user-images.githubusercontent.com/36727740/79844884-95e31900-83bc-11ea-8bf8-7b3173c01d85.png)  
This is how it should be displayed
![image](https://user-images.githubusercontent.com/36727740/79844999-bc08b900-83bc-11ea-8b9d-040b91eb79a0.png)

## Environment (please complete the following information)

- Operating System: macOS, but tested on Windows also
- Browser, Version Chrome 80.0.3987.149
- React-Bootstrap Version 1.0.0-beta.16, but is reproducible on 1.0.0

**Comment by taion** [comment]
I don't think we want to always recalculate the popper position on content changes, as that could be quite expensive, and most changes won't reposition the menu.

@jquense does it make sense to expose the `scheduleUpdate` callback from `usePopper` here somehow?

**Comment by lukezavrel** [comment]
I see, so is there currently a way to force popper to recalculate the position? That would be enough for my case

**Comment by jac-bah** [comment]
YES PLEASE!  I am running into the same issue and cannot figure out how to get a reference to the dropdown's popper instance to call scheduleUpdate.  I can wrap a <Popover> with a custom forwardRef to access to props.popper, but that didn't work for <Dropdown>.  Exposing scheduleUpdate directly would be a fantastic option!

**Comment by jithin-quintet** [comment]
As the scheduleUpdate/update is not exposed, a hacky way is to trigger a resize when the items are changed, so that popper will position it correctly.

https://codesandbox.io/s/laughing-newton-6ived
